src/typings.ts

```diff
@@ -82,12 +85,13 @@ export interface WalletPosition {
 }
 
 export interface NetworkConfig {
+  AUTH_URL: string
```

Comment:
Is the auth url optional or required?

Reply:
`required`, the whole new Auth flow depends on it.

Reply:
`AUTH_URL` is basically the wallet verify url